### PR TITLE
Support for read-only mode

### LINKED ISSUE #53: Introduce [disabled] property
Both integrations (Angular and Vue) allows changing the editor to read-only mode.

In React you can do it by saving the `editor` from `onInit()` callback. If we will introduce the `[disabled]` property, switching between those modes should be easier.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Introduced the `[disabled]` property which allows switching the editor to read-only mode. Closes #53.

### CODE REVIEW

File: tests/ckeditor.jsx [hunk 1]
```diff
@@ -372,6 +372,64 @@ describe( 'CKEditor Component', () => {
 				} );
 			} );
 		} );
+
+		describe.only( '#disabled', () => {
```
Comment (pomek) [inline tests/ckeditor.jsx]: ❤️

**Comment by pomek** [comment]
Because https://github.com/ckeditor/ckeditor5/pull/1364 is still open, I added the new property in mentioned PR.

File: src/ckeditor.jsx [hunk 2]
```diff
@@ -103,11 +127,13 @@ CKEditor.propTypes = {
 	onChange: PropTypes.func,
 	onInit: PropTypes.func,
 	onFocus: PropTypes.func,
-	onBlur: PropTypes.func
+	onBlur: PropTypes.func,
+	disabled: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.string ] )
```
Comment (ma2ciek) [inline src/ckeditor.jsx]: I don't agree here. Why we should support multiple types here? Can't we just support only `boolean`?

Comment (pomek) [reply src/ckeditor.jsx]: Why not? You can do `disabled="true"` (as I did before I discovered `disabled={true}`) and it still will work.

Comment (ma2ciek) [reply src/ckeditor.jsx]: But `disabled="true"` isn't the preferred way to pass a boolean option to react component as you find out yourself... IMO we should support more than one type if there're a concept and more logic behind the other types (like string and array of strings).

Comment (pomek) [reply src/ckeditor.jsx]: Now the component will accept boolean value only.

File: sample/index.html [hunk 3]
```diff
@@ -51,6 +56,16 @@ <h2>Sample</h2>
 			} ),
 			document.getElementById( 'root' )
 		);
+
+		document.getElementById( 'readOnly' ).addEventListener( 'click', function() {
+			window.editor.isReadOnly = !window.editor.isReadOnly;
```
Comment (ma2ciek) [inline sample/index.html]: What if the editor isn't available here? 😄

Comment (pomek) [reply sample/index.html]: I moved this code to `onInit` component's callback.

**Pull request merged**